components/gantt/gantt-tree/editing/popup.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Popup Editing
+page_title: Gantt Tree - Popup Editing
+description: Popup editing of data in Gantt Tree for Blazor.
+slug: gant-tree-popup-editing
+tags: telerik,blazor,gantt,popup,editing
+published: True
+position: 15
+---
+
+# Gantt Tree Popup Editing
+
+Popup editing lets the user click an [Edit command button]({%slug components/grid/columns/command%}) on the row, and a popup shows up with all its editable columns open up for changes. They can then click the `Save` button in the dialog to submit the changes to the model. This fires the `OnUpdate` event where your code receives the updated model so you can work with the data (for example, to call the appropriate method of your service).
```

Comment:
The current behavior of the popup form is to always render the 4 properties associated with a task (name, percentage, start, end), thus it will not render editors for any additional columns(or omit an editor if one of those fields is not defined within a column). 
I've added a PR with a clarification for this, but it might be good to include it here as well. https://github.com/telerik/blazor-docs/pull/805